src/batch/src/error.rs

```diff
@@ -139,6 +139,13 @@ pub enum BatchError {
 
     #[error("Not enough memory to run this query, batch memory limit is {0} bytes")]
     OutOfMemory(u64),
+
+    #[error(transparent)]
```

Comment:
Should give more context here

```suggestion
    #[error("Failed to spill out to disk")]
```